rosetta/create-t5x-distribution.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
```

Comment:
Do we want to add argument parsing to this script like those in [install-t5x.sh](https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/container/install-t5x.sh)?

Reply:
I don't mind adding to be consistent with the other scripts. Is that your recommendation?

Reply:
Yes, I would recommend so. I see the script relies on certain variables that are currently passed externally via environment variables. Allowing them to be set via the command line can greatly improve readability and maintainability.

Reply:
I've added switches and updated the docs